gradle-module/src/main/java/org/example/Main.java

```diff
@@ -22,4 +23,11 @@ public static void main(String[] args) {
     private static void test() {
         System.out.println("test");
     }
-}
\ No newline at end of file
+
+   /* asd
+    we
+    asd
+
+    */
+    
```

Comment:
[Category: Maintainability] [Severity: Medium] The comment block at the end of the file appears to be a placeholder and does not provide meaningful information. Consider removing it or replacing it with relevant documentation. Rationale: Clear and relevant comments enhance code maintainability and understanding for future developers.